### PR TITLE
Fix binding consumer: also bind if no routing keys are passed

### PR DESCRIPTION
**Pull request by mweibel**

### CODE REVIEW

File: src/AmqpBundle/Factory/ConsumerFactory.php [hunk 1]
```diff
@@ -109,9 +109,12 @@ public function get($class, $connexion, array $exchangeOptions, array $queueOpti
         // Declare the queue
         $queue->declareQueue();
 
-        // Bind the queue to some routing keys
-        foreach ($queueOptions['routing_keys'] as $routingKey) {
-            $queue->bind($exchangeOptions['name'], $routingKey);
+        if(count($queueOptions['routing_keys'])) {
```
Comment (lsmith77) [inline src/AmqpBundle/Factory/ConsumerFactory.php]: missing space after ``if``

Comment (mweibel) [inline src/AmqpBundle/Factory/ConsumerFactory.php]: fixed, thanks

**Comment by fabdsp** [comment]
Thx for this PR !
But why do you want to bind a queue without routing keys ?

**Comment by mweibel** [comment]
Routing keys are optional when [binding a queue to an exchange](https://www.rabbitmq.com/tutorials/amqp-concepts.html#bindings):

> Bindings are rules that exchanges use (among other things) to route messages to queues. To instruct an exchange E to route messages to a queue Q, Q has to be bound to E. Bindings may have an optional routing key attribute used by some exchange types. The purpose of the routing key is to select certain messages published to an exchange to be routed to the bound queue. In other words, the routing key acts like a filter. 

We don't use routing keys, therefore everything which is published on the exchange gets routed to the queue attached (and we don't publish with routing keys either).

**Comment by fabdsp** [comment]
ok !
2.1.0

**Comment by mweibel** [comment]
Great, thanks!

**Pull request merged**